plugins/cicd-statistics/api-report.md

```diff
@@ -0,0 +1,182 @@
+//
+// @public
+export type BuildWithRaw<T = any> = Build & {
+  raw: T;
+};
+
+// Warning: (ae-missing-release-tag) "CicdConfiguration" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdConfiguration {
+  availableStatuses: ReadonlyArray<FilterStatusType>;
+  defaults: Partial<CicdDefaults>;
+  formatStageName: (parentNames: Array<string>, stageName: string) => string;
+}
+
+// Warning: (ae-missing-release-tag) "CicdDefaults" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdDefaults {
+  collapsedLimit: number;
+  // (undocumented)
+  filterStatus: Array<FilterStatusType>;
+  // (undocumented)
+  filterType: FilterBranchType<'all'>;
+  lowercaseNames: boolean;
+  normalizeTimeRange: boolean;
+  // (undocumented)
+  timeFrom: Date;
+  // (undocumented)
+  timeTo: Date;
+}
+
+// Warning: (ae-missing-release-tag) "CicdState" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdState {
+  // (undocumented)
+  builds: Array<Build>;
+}
+
+// Warning: (ae-missing-release-tag) "CicdStatisticsApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface CicdStatisticsApi {
+  // (undocumented)
+  fetchBuilds(options: FetchBuildsOptions): Promise<CicdState>;
+  // (undocumented)
+  getConfiguration(): Promise<Partial<CicdConfiguration>>;
+}
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsApiRef" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsApiRef: ApiRef<CicdStatisticsApi>;
+
+// Warning: (ae-missing-release-tag) "cicdStatisticsPlugin" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const cicdStatisticsPlugin: BackstagePlugin<
+  {
+    entityContent: RouteRef<undefined>;
+  },
+  {}
+>;
+
+// Warning: (ae-forgotten-export) The symbol "EntityPageCicdCharts" needs to be exported by the entry point index.d.ts
+// Warning: (ae-missing-release-tag) "EntityCicdStatisticsContent" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export const EntityCicdStatisticsContent: EntityPageCicdCharts;
+
+// Warning: (ae-missing-release-tag) "FetchBuildsOptions" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public
+export interface FetchBuildsOptions {
+  // (undocumented)
+  abortSignal: AbortSignal;
+  // (undocumented)
+  entity: Entity;
+  // (undocumented)
+  filterStatus: Array<FilterStatusType<'all'>>;
```

Comment:
Think `FilterStatusType | 'all'` would be a nicer pattern? I had to jump to the type to figure out what it was doing

Reply:
Yes this is a bit iffy agreed.